multithreading.c

```diff
@@ -54,6 +54,7 @@ void *thr_func(void *arg) {
     }
   }
 
+  data->reentrantSeed = seed;
```

Comment:
Just a reminder for myself: I don't need the seed in the struct, i can remove it from the struct